specification/postgresql/resource-manager/Microsoft.DBforPostgreSQL/2017-04-30-preview/postgresql.json

```diff
@@ -2,7 +2,7 @@
   "swagger": "2.0",
   "info": {
     "title": "PostgreSQLManagementClient",
-    "description": "The Microsoft Azure management API provides create, read, update, and delete functionality for Azure PostgreSQL resources including servers, databases, firewall rules, log files and configurations.",
+    "description": "The Microsoft Azure management API provides create, read, update, and delete functionality for Azure PostgreSQL resources including servers, databases, firewall rules, VNET rules, log files and configurations.",
```

Comment:
This JSON is also invalid: SyntaxError: Unexpected token ] in JSON at position 14605
    at Object.parseJson (/home/travis/build/Azure/azure-rest-api-specs/node_modules/oav/lib/util/utils.js:78:15)
    at SpecValidator.initialize (/home/travis/build/Azure/azure-rest-api-specs/node_modules/oav/lib/validators/specValidator.js:91:20)
    at Object.validateSpec (/home/travis/build/Azure/azure-rest-api-specs/node_modules/oav/lib/validate.js:60:20)
    at Context.<anonymous> (/home/travis/build/Azure/azure-rest-api-specs/test/semantic.js:19:44)
    at callFn (/home/travis/build/Azure/azure-rest-api-specs/node_modules/mocha/lib/runnable.js:348:21)
    at Test.Runnable.run (/home/travis/build/Azure/azure-rest-api-specs/node_modules/mocha/lib/runnable.js:340:7)
    at Runner.runTest (/home/travis/build/Azure/azure-rest-api-specs/node_modules/mocha/lib/runner.js:443:10)
    at /home/travis/build/Azure/azure-rest-api-specs/node_modules/mocha/lib/runner.js:549:12
    at next (/home/travis/build/Azure/azure-rest-api-specs/node_modules/mocha/lib/runner.js:361:14)
    at /home/travis/build/Azure/azure-rest-api-specs/node_modules/mocha/lib/runner.js:371:7
    at next (/home/travis/build/Azure/azure-rest-api-specs/node_modules/mocha/lib/runner.js:295:14)
    at Immediate.<anonymous> (/home/travis/build/Azure/azure-rest-api-specs/node_modules/mocha/lib/runner.js:339:5)
    at runCallback (timers.js:781:20)
    at tryOnImmediate (timers.js:743:5)
    at processImmediate [as _immediateCallback] (timers.js:714:5)

Reply:
fixed